actionpack/lib/action_dispatch/middleware/templates/rescues/blocked_host.text.erb

```diff
@@ -1,5 +1,5 @@
 Blocked host: <%= @host %>
 
-To allow requests to <%= @host %>, add the following to your environment configuration:
+To allow requests to <%= @host %> make sure it is a valid hostname (containing only numbers, letters, dashes and dots), then add the following to your environment configuration:
```

Comment:
We also need to update the HTML version of this.

Reply:
Oops, yep.